### ISSUE TITLE
Feature Request: Make all metadata in Reader available to Obsidian Export

### ISSUE THREAD

**Issue by jaxley** [readwiseio/obsidian-readwise#51]
The existing Obsidian export is pretty cool and I like the flexibility of the template customization. What would be even better is if the other metadata that is in Readwise is exposed to Obsidian templates. I want to generate citations for articles, for example, but I can't because publish date and saved date.

The summary would be really useful to have exported for searching, etc.
Progress would be nice if you wanted to keep track of media consumption progress.

**Comment by FeralFlora** [comment]
Yes, I came here looking for this! It seems so obvious that users would want all of this missing data available - especially the ones you mention, @jaxley.

I also find that the Reader data doesn't always match what's imported into Obsidian. For example, the author is sometimes just the domain name, even though the Reader metadata has the actual name of the author.
